dist/gridlex.css

```diff
@@ -509,116 +514,116 @@
-  margin-left: 8.33333%; }
-
-[class~="grid"] > [data-push-left*="off-2"],
-[class*="grid-"] > [data-push-left*="off-2"],
-[class*="grid_"] > [data-push-left*="off-2"] {
-  margin-left: 16.66667%; }
-
-[class~="grid"] > [data-push-left*="off-3"],
-[class*="grid-"] > [data-push-left*="off-3"],
-[class*="grid_"] > [data-push-left*="off-3"] {
-  margin-left: 25%; }
-
-[class~="grid"] > [data-push-left*="off-4"],
-[class*="grid-"] > [data-push-left*="off-4"],
-[class*="grid_"] > [data-push-left*="off-4"] {
-  margin-left: 33.33333%; }
-
-[class~="grid"] > [data-push-left*="off-5"],
-[class*="grid-"] > [data-push-left*="off-5"],
-[class*="grid_"] > [data-push-left*="off-5"] {
-  margin-left: 41.66667%; }
-
-[class~="grid"] > [data-push-left*="off-6"],
-[class*="grid-"] > [data-push-left*="off-6"],
-[class*="grid_"] > [data-push-left*="off-6"] {
-  margin-left: 50%; }
-
-[class~="grid"] > [data-push-left*="off-7"],
-[class*="grid-"] > [data-push-left*="off-7"],
-[class*="grid_"] > [data-push-left*="off-7"] {
-  margin-left: 58.33333%; }
-
-[class~="grid"] > [data-push-left*="off-8"],
-[class*="grid-"] > [data-push-left*="off-8"],
-[class*="grid_"] > [data-push-left*="off-8"] {
-  margin-left: 66.66667%; }
-
-[class~="grid"] > [data-push-left*="off-9"],
-[class*="grid-"] > [data-push-left*="off-9"],
-[class*="grid_"] > [data-push-left*="off-9"] {
-  margin-left: 75%; }
-
-[class~="grid"] > [data-push-left*="off-10"],
-[class*="grid-"] > [data-push-left*="off-10"],
-[class*="grid_"] > [data-push-left*="off-10"] {
-  margin-left: 83.33333%; }
-
-[class~="grid"] > [data-push-left*="off-11"],
-[class*="grid-"] > [data-push-left*="off-11"],
-[class*="grid_"] > [data-push-left*="off-11"] {
-  margin-left: 91.66667%; }
-
 [class~="grid"] > [data-push-right*="off-0"],
 [class*="grid-"] > [data-push-right*="off-0"],
 [class*="grid_"] > [data-push-right*="off-0"] {
   margin-right: 0; }
 
+[class~="grid"] > [data-push-left*="off-1"],
+[class*="grid-"] > [data-push-left*="off-1"],
+[class*="grid_"] > [data-push-left*="off-1"] {
+  margin-left: 8.33333%; }
+
 [class~="grid"] > [data-push-right*="off-1"],
 [class*="grid-"] > [data-push-right*="off-1"],
 [class*="grid_"] > [data-push-right*="off-1"] {
   margin-right: 8.33333%; }
 
+[class~="grid"] > [data-push-left*="off-2"],
+[class*="grid-"] > [data-push-left*="off-2"],
+[class*="grid_"] > [data-push-left*="off-2"] {
+  margin-left: 16.66667%; }
+
 [class~="grid"] > [data-push-right*="off-2"],
 [class*="grid-"] > [data-push-right*="off-2"],
 [class*="grid_"] > [data-push-right*="off-2"] {
   margin-right: 16.66667%; }
 
+[class~="grid"] > [data-push-left*="off-3"],
+[class*="grid-"] > [data-push-left*="off-3"],
+[class*="grid_"] > [data-push-left*="off-3"] {
+  margin-left: 25%; }
+
 [class~="grid"] > [data-push-right*="off-3"],
 [class*="grid-"] > [data-push-right*="off-3"],
 [class*="grid_"] > [data-push-right*="off-3"] {
   margin-right: 25%; }
 
+[class~="grid"] > [data-push-left*="off-4"],
```

Comment:
I have not look at all your work, but I see that offset are generated twice. Need to be fix before I decide to accept your great work :)

Reply:
I checked the diff of the CSS files again and it seems that it is not generated twice, but the order of rules between offset left and offset right is different if I put them in one for loop (as it is in this PR). If I seperate the loops as given below, the diff does not show any differences anymore. Would you like the loops to be seperate in order to have less differences?

    // Function to generate the offset
    @mixin makeOff($off)
    {
      [#{$gl-attributeName}~="#{$gl-gridName}"],
      [#{$gl-attributeName}*="#{$gl-gridName}-"],
      [#{$gl-attributeName}*="#{$gl-gridName}_"] {
        @for $i from 0 through $gl-colCount - 1 {
          > [data-push-left*="#{$off}-#{$i}"]{ margin-left: map-get($gl-colPortions, $i); }
        }
      }
    
      [#{$gl-attributeName}~="#{$gl-gridName}"],
      [#{$gl-attributeName}*="#{$gl-gridName}-"],
      [#{$gl-attributeName}*="#{$gl-gridName}_"] {
        @for $i from 0 through $gl-colCount - 1 {
          > [data-push-right*="#{$off}-#{$i}"]{ margin-right: map-get($gl-colPortions, $i); }
        }
      }
    }